debezium-core/src/main/java/io/debezium/pipeline/source/snapshot/incremental/AbstractIncrementalSnapshotChangeEventSource.java

```diff
@@ -620,6 +622,8 @@ private List<String> expandDataCollectionIds(List<String> dataCollectionIds) {
                     return ids.isEmpty() ? Stream.of(x) : ids.stream();
                 })
                 .collect(Collectors.toList());
```

Comment:
I think you can simplify it using `collect( Collectors.toCollection( LinkedHashSet::new ));`

Can also try to add a test?

Reply:
@mfvitale distinct seems to be even better solution, since it preserves the insertion order